WorkInProgress/Administration/xASL_adm_PreparePediatricTPM.m

```diff
@@ -186,7 +186,7 @@
 	end
 
 	% Copy all the adult masks and transform to pediatric size
-	listMask = {'brainmask' 'TotalGM' 'TotalWM' 'WholeBrain'  'brainmask_supratentorial' 'ParenchymNarrow' 'LeftRight' 'DeepWM' 'CentralWM_QC' 'GhostSignalRatio' 'MNI_Structural'};
+	listMask = {'brainmask' 'WholeBrain'  'Supratentorial_GM_WM' 'ParenchymNarrow' 'LeftRight' 'DeepWM' 'CentralWM_QC' 'MNI_Structural'};
```

Comment:
Remove totalWM from line 222.

Reply:
Fixed in 0741136d0b23ba106e320d13c8aeae8e92dd4910